README.md

```diff
@@ -29,6 +29,7 @@ https://annu12340.github.io/Javascript-Notes/
 
 <br/>
 
+- Local storage vs Session storage
```

Comment:
Could add this heading on line 29 (because after that <br/> on line 30, includes more advanced topics)

Reply:
okay, just heading or the whole definition part